theano/scan_module/tests/test_scan.py

```diff
@@ -5529,3 +5529,41 @@ def count_up():
 
         _, updates = theano.scan(count_up, n_steps=20)
         func = theano.function(inputs=[inc], outputs=[], updates=updates)
+
+
+class TestGradUntil(unittest.TestCase):
+
+    def setUp(self):
+        self.x = tensor.vector(name='x')
+        self.until = tensor.scalar(name='until', dtype='int64')
```

Comment:
I don't like the name until here. I don't have a much better name. The only thing that come to mind is cond, but it isn't the full condition. What about threshold?